Tasks/AzureFileCopyV1/task.json

```diff
@@ -13,7 +13,7 @@
     "version": {
         "Major": 1,
         "Minor": 0,
-        "Patch": 140
+        "Patch": 141
```

Comment:
We will not need patch update if you take my comment below on initialize function.